shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -336,6 +338,22 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
       // Reset for the next pass:
       flatland_layer_index++;
     }
+
+    // Set up the input interceptor at the top of the scene, if applicable.  It
+    // will capture all input, and any unwanted input will be reinjected into
+    // embedded views.
+    if (intercept_all_input_) {
+      fuchsia::ui::composition::TransformId input_interceptor_node =
+          flatland_->NextTransformId();
+      flatland_->flatland()->CreateTransform(input_interceptor_node);
+      flatland_->flatland()->AddChild(root_transform_id_,
+                                      input_interceptor_node);
+      flatland_->flatland()->SetHitRegions(
```

Comment:
Let's retain the overflow warning here too.

Reply:
Done